src/app/pages/storage/volumes/volume-import-wizard/volume-import-wizard.component.ts

```diff
@@ -251,27 +252,19 @@ export class VolumeImportWizardComponent {
   preInit(entityWizard: EntityWizardComponent) {
     this.productType = window.localStorage.getItem('product_type');
     if (this.productType.includes('SCALE')) {
-      this.wizardConfig.splice(1,1);
-      this.importIndex = 1;
+      this.wizardConfig.splice(0,2);
+      this.importIndex = 0;
+      this.getImportableDisks();
     }
   }
 
   afterInit(entityWizard: EntityWizardComponent) {
-    const createPoolText = T("Create Pool")
+    const nextText = T("Next ")
```

Comment:
Good thinking here; however, the entity wizard (our pattern for making wizards) will default to 'Next', so you can just remove this line (#262) and also #264, which sets the customNext text.  And one more thing: Our boss doesn't like us to leave console logs, so please remove #267 as well.